src/views/CapturesView.test.js

```diff
@@ -0,0 +1,295 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { act, render, screen, within, cleanup } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import theme from '../components/common/theme';
+import { ThemeProvider } from '@material-ui/core/styles';
+import { AppProvider } from '../context/AppContext';
+import { CapturesContext } from '../context/CapturesContext';
+import { GrowerContext } from '../context/GrowerContext';
+import { SpeciesContext } from '../context/SpeciesContext';
+import { TagsContext } from '../context/TagsContext';
+import CaptureView from './CapturesView';
+import {
+  CAPTURE,
+  CAPTURES,
+  GROWER,
+  ORGS,
+  TAG,
+  TAGS,
+  SPECIES,
+  capturesValues,
+  growerValues,
+  tagsValues,
+  speciesValues,
+} from '../components/tests/fixtures';
+
+import * as loglevel from 'loglevel';
+const log = loglevel.getLogger('../tests/verify.test');
+
+jest.setTimeout(7000);
+jest.mock('../api/growers');
+jest.mock('../api/treeTrackerApi');
+
+describe.skip('Captures View', () => {
+  // mock the treeTrackerApi
+  let captureApi = require('../api/treeTrackerApi').default;
+
+  const getCaptures = () => {
+    log.debug('mock getCaptures:');
```

Comment:
I don't think debug logs are needed in tests and can be removed. They could also slow down tests as well.